pkg/volume/util/volumepathhandler/volume_path_handler_linux.go

```diff
@@ -131,6 +131,12 @@ func parseLosetupOutputForDevice(output []byte, path string) (string, error) {
 		return "", errors.New(ErrDeviceNotFound)
 	}
 
+	// Fully expand any symlinks in the device path, losetup output has them expanded too
+	realPath, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return "", fmt.Errorf("failed to evaluate path %s: %s", path, err)
+	}
```

Comment:
should we instead call `losetup` on expanded path, so as if behaviour of losetup changes in future - we don't break anything?

Reply:
I think evaluating paths before calling losetup is not useful. But I see your point, I added check for both `realPath` and original `path`, just in case.

Reply:
> I think evaluating paths before calling losetup is not useful.

Can you explain why is it not useful? Do you foresee any breakage? From very afar it sounds like it is better to pass expanded path to losetup and scan for only one path rather than two?

Reply:
The initial concern was that losetup can change in the future and it may not expand symlinks in `losetup -j` output. So if it lists symlinks instead of evaluating them, we should check for the unevaluated path too.